lib/active_admin_sidebar/activeadmin_views_pages_base.rb

```diff
@@ -2,7 +2,7 @@ class ActiveAdmin::Views::Pages::Base < Arbre::HTML::Document
 
   def build_page_content
     build_flash_messages
-    div id: "active_admin_content", class: main_content_classes do
+    div id: 'active_admin_content', class: main_content_classes do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.